src/wheelofhunger/controllers/WOH_Insert.java

```diff
@@ -0,0 +1,100 @@
+package wheelofhunger.controllers;
+
+import java.io.IOException;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Servlet implementation class WOH_Insert
+ */
+@WebServlet("/Insert")
+public class WOH_Insert extends HttpServlet {
+	private static final long serialVersionUID = 1L;
+	static String             url              = "jdbc:mysql://wheelofhunger.ddns.net:3306/WOHDB";
+	static String             user             = "remote";
+	static String             password         = "1234";
+	static Connection         connection       = null;
+       
+    /**
+     * @see HttpServlet#HttpServlet()
+     */
+    public WOH_Insert() {
+        super();
+        // TODO Auto-generated constructor stub
+    }
+
+	/**
+	 * @see HttpServlet#doGet(HttpServletRequest request, HttpServletResponse response)
+	 */
+	protected void doGet(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		// TODO Auto-generated method stub
+		response.getWriter().append("Served at: ").append(request.getContextPath());
+	}
+
+	/**
+	 * @see HttpServlet#doPost(HttpServletRequest request, HttpServletResponse response)
+	 */
+	protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+		// TODO Auto-generated method stub
+		response.setContentType("text/html;charset=UTF-8");
+		String name = request.getParameter("name");
+	    int price = -1;
+	    int type = -1;
+    	try{
+    		price = Integer.parseInt(request.getParameter("price"));
+    		type = Integer.parseInt(request.getParameter("type"));
+    	}catch(Exception e){
+    		System.out.println("fields are null");
+    	}
+	    
+    	String cusines = request.getParameter("cusines");
```

Comment:
Cuisines must be in the form of a comma separated string that has to be built on the front end.  Same thing with Notes but is should be separated by semicolons.